app/views/membership_applications/information.html.haml

```diff
@@ -0,0 +1,14 @@
+- if current_user.is_member == true
+  %header.entry-header
+    %h1.entry-title Såhär annonserar du ditt medlemsskap
+  .post-title-divider
+    %span
+  .entry-content
+
+
+- else
+  %header.entry-header
```

Comment:
There's too much repeated code.  You need to DRY this up.
The only thing that's different is the text in the`%h1.entry-title`  line, so you should have the conditional affect only that line.  You could use a helper:
`helpers/membership_application_helper.rb`

    def user_or_member_instructions(user)
      user.is_member? ? 'Såhär annonserar du ditt medlemsskap' : 'Såhär ansöker du om medlemsskap'
    end

`%h1.entry-title user_or_member_instructions(current_user)`

We know that there will be different content in the body of this per PT: [Landing page for member with complete  company](https://www.pivotaltracker.com/story/show/135009145), so you'll need to do more conditionals for the body.  With this design, I'd suggest using  partials (perhaps with `content_for:`) and a helper that returns which partial to show based on the role

If you separate the roles so that you have views/user and views/member, you can use the same layout but populate the header and the body with different contents.  It will be cleaner than having the conditionals in this file or in helpers that it calls.   ]